### PR TITLE
Updates pot file

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

File: i18n/po/template.pot [hunk 1]
```diff
@@ -3104,6 +3221,14 @@ msgstr ""
 msgid "Your bitcoin wallet is ready!"
 msgstr ""
 
+#: www/views/modals/chooseFeeLevel.html:38
+msgid "Your could not set a fee higher than 1000 satoshis/byte."
```
Comment (JDonadio) [inline i18n/po/template.pot]: Typo: "your"

File: i18n/po/template.pot [hunk 2]
```diff
@@ -3104,6 +3221,14 @@ msgstr ""
...
+msgstr ""
+
+#: www/views/modals/chooseFeeLevel.html:34
+msgid "Your fee is lower than the recommended super economy fee ({{getMinimumRecommeded()}} satoshis/byte). Your transaction may never get confirmed."
```
Comment (JDonadio) [inline i18n/po/template.pot]: Should be: "lower than recommended..." and "The transaction may never get confirmed"

File: i18n/po/template.pot (hunk 1 above)
Comment (cmgustavo) [inline i18n/po/template.pot]: Thanks!

File: i18n/po/template.pot (hunk 2 above)
Comment (cmgustavo) [inline i18n/po/template.pot]: Thanks!

**Comment by cmgustavo** [comment]
Updated.

**Pull request merged**